src/webview/src/components/fbaCheckbox.js

```diff
@@ -202,23 +222,79 @@ export default function FBACheckbox(props) {
 
     const backgroundFragments = (
         <React.Fragment>
-            <DialogContentText paragraph>
-                <TextField name='backgroundDescription' onChange={handleValueChanges} margin="dense" id={'backgroundDescription-field-' + props.name}
-                    InputLabelProps={{ shrink: true, }} fullWidth multiline value={values.backgroundDescription}
-                    label='Background'
-                    placeholder='Please enter some background information on this root cause.' />
-            </DialogContentText>
+            <Divider variant="middle" />
+            <Accordion>
+                <AccordionSummary
```

Comment:
does this provide a few lines of Summary already here/always? 
I'd prefer if user can always see (some lines, best case scrollable) from the description while adding/writing the comments. (otherwise they constantly have to switch). 
Do we run out of space? (we can extend the FBACheckbox dialog to fullscreen if needed)

Reply:
It was only meant for a better overview in case of very large content, instead I've changed it now to be extended by default. One can close it instead on demand.